downloadIstio.sh

```diff
@@ -21,11 +21,41 @@ else
   # TODO we should check more/complain if not likely to work, etc...
   OSEXT="linux"
 fi
+
```

Comment:
Running programs downloaded from the net is one of the things I tell my kids to never do - IMHO the script should
just be removed, not extended. Making it change the user config automatically is making it even more intrusive.
The real solution is to publish rpm/deb/dmg installers - istioctl and dump_clusters are the only executable components,
and for them it might be safer to use docker images.

Reply:
Yes you’re right, it’s just a temporary solution. 
The real solution for me will be a istioctl installer (rpm/deb/dmg) and a Istio helm chart repo. But we don’t have it. 
Is it in the roadmap?

Reply:
We do have a .deb file - including istioctl, envoy, agent, etc.
We use fpm - so could generate rpm/dmg/etc - but someone would need to test,etc.